api/room/schema.py

```diff
@@ -183,7 +185,29 @@ def mutate(self, info, room_id, **kwargs):
         return DeleteRoom(room=exact_room)
 
 
+class UpdateFirebaseToken(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int(required=True)
+        firebase_token = graphene.String(required=True)
+    room = graphene.Field(Room)
+
+    def mutate(self, info, room_id, **kwargs):
+        MRM_PUSH_URL = os.getenv("MRM_PUSH_URL")
```

Comment:
Please use the config.py file to access environment vars then rebase so we can merge this.